### PR TITLE
Remove the Website from README.md

### PR DESCRIPTION
**Pull request by longstone**
remove the website, seems to be fraud

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,8 +19,8 @@ Wow, well, loads. Here is a quick list.
 * Loads of code removed for a faster, more reliable PhotoShow. Hopefully.
 * Uploaders have lost a load of rights. Basically, all they can do is upload. No file removing anymore.
 * Rights management SHOULD be the same as before. But, just in case, please check after updating
-* Updated phpThumb library
-* Added configuration values for `quality_small` and `quality_mini` thumbnails with default of `90` - change them through your own config.php, like `$config->quality_small = 90;`  
+* Updated phpThumb library
```
Comment (LudovicRousseau) [inline README.md:22]: Why this other documentation change?

Comment (longstone) [reply README.md:22]: This Happyend accidentaly. I used the github Web editor :/

**Comment by kangarooo** [comment]
#380 this will fix it and add screenshots.

**Comment by longstone** [comment]
closed as #380 also resolves this issue

**Comment by kangarooo** [comment]
This isnt pulled yet, so how then its resolved? @longstone

**Comment by longstone** [comment]
@kangarooo I hope one day @LudovicRousseau has time to check again. I closed this PR to reduce open PRs. As in #380 the url is removed and screenshots are added, for me it will be better to merge that instead of this.

**Comment by LudovicRousseau** [comment]
@kangarooo what do you want me to do exactly?
I am not the owner of this repository. I can't merge the patch. I was just asking a question.

I am not sure @thibaud-rohmer is still active on his project. Time to fork it?

**Comment by longstone** [comment]
> @kangarooo what do you want me to do exactly?
> I am not the owner of this repository. I can't merge the patch. I was just asking a question.
> 
> I am not sure @thibaud-rohmer is still active on his project. Time to fork it?

@LudovicRousseau: My bad. I tought you have rights. +1 Time to fork!

**Pull request closed without merging**